src/app.js

```diff
@@ -29,9 +43,12 @@ function App() {
             <Route exact path="/">
               <Redirect push to="/login" />
             </Route>
-            <UnauthenticatedRoute path="/login" component={LoginRoute} />
-            <UnauthenticatedRoute path="/signup" component={SignupRoute} />
-            <ProtectedRoute path="/dashboard" component={DashboardRoute} />
+            <UnauthenticatedRoute path={LoginRoute.path} component={LoginRoute} />
+            <UnauthenticatedRoute path={SignupRoute.path} component={SignupRoute} />
+            <ProtectedRoute path={DashboardRoute.path} component={DashboardRoute} />
+            <ProtectedRoute path={NudgeNewRoute.path} component={NudgeNewRoute} />
+            <ProtectedRoute path={NudgeUpdateRoute.path} component={NudgeUpdateRoute} />
+            <ProtectedRoute path={NudgeDeleteRoute.path} component={NudgeDeleteRoute} />
```

Comment:
NIT: I usually just create another file called "router.js" and do all the routes there just so it's separate